packages/modules/viewer-react/package.json

```diff
@@ -2,9 +2,14 @@
   "name": "@itwin/viewer-react",
   "version": "5.0.0-dev.7",
   "description": "Configurable iTwin.js viewer",
-  "main": "lib/cjs/index.js",
+  "type": "module",
   "module": "lib/esm/index.js",
```

Comment:
Same as in other package. I think `"module": "lib/esm/index.js"` can be removed and `lib/esm` can be changed to `lib`.